bundles/binding/org.openhab.binding.zwave/database/everspring/se812.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>SE812</Model>
+	<Label lang="en">Indoor Siren</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x25</id></Class>
+		<Class><id>0x71</id></Class>
+		<Class><id>0x85</id></Class>
+		<Class><id>0x80</id></Class>
+		<Class><id>0x72</id></Class>
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Configuration>
+	</Configuration>
+	<Associations>
+		<Group>
+			<Index>1</Index>
+			<Maximum>5</Maximum>
+			<Label lang="en">Control/Report Group</Label>
```

Comment:
If this group is to be used to send notifications to the controller, please can you add -:
```
<SetToController>true</SetToController>
```
to the group.